src/olympia/addons/serializers.py

```diff
@@ -703,6 +704,36 @@ def to_representation(self, obj):
         return data
 
 
+class VersionBasketSerializer(SimpleVersionSerializer):
+    class Meta:
+        model = Version
+        fields = ('id', 'compatibility', 'is_strict_compatibility_enabled',
+                  'version')
```

Comment:
Does basket need any of this?  `is_strict_compatibility_enabled` is a weird thing to send.

Reply:
I erred on the side of caution, figured since they were interested in compatibility info, they might want this as well.